concepticondata/conceptlists/DellaRosa-2010-417.tsv

```diff
@@ -0,0 +1,418 @@
+DellaRosa-2010-417-70	70	Fervore	fervour			209	224	341	555	345	559	556
+DellaRosa-2010-417-71	71	Fiducia	confidence			232	230	587	370	436	502	495
+DellaRosa-2010-417-72	72	Filosofia	philosophy			179	206	547	521	318	519	438
+DellaRosa-2010-417-73	73	Follia	madness			319	380	538	447	522	475	515
+DellaRosa-2010-417-74	74	Forza	force	2526	POWER	416	388	563	258	471	413	400
+DellaRosa-2010-417-75	75	Fremito	thrill			335	312	273	516	304	390	600
+DellaRosa-2010-417-76	76	Frenesia	frenzy			250	300	353	600	347	518	581
+DellaRosa-2010-417-77	77	Funzione	function			273	274	414	420	341	465	592
+DellaRosa-2010-417-78	78	Furia	fury			256	379	360	430	354	405	526
+DellaRosa-2010-417-79	79	Furto	theft			455	456	518	333	575	282	444
+DellaRosa-2010-417-80	80	Futuro	future	2427	FUTURE	200	235	611	375	297	629	563
+DellaRosa-2010-417-81	81	Gentilezza	kindness	3468	KINDNESS	300	409	579	279	506	494	350
+DellaRosa-2010-417-82	82	Gioia	joy	2524	JOY	313	420	574	278	504	467	460
+DellaRosa-2010-417-83	83	Giudizio	judgement	1067	JUDGMENT	250	279	533	415	358	567	559
+DellaRosa-2010-417-84	84	Giuramento	oath	1712	OATH	309	344	393	400	417	400	553
+DellaRosa-2010-417-85	85	Giustificazione	justification			279	260	488	362	450	480	621
+DellaRosa-2010-417-86	86	Giustizia	justice			245	306	568	388	470	535	400
+DellaRosa-2010-417-87	87	Ideale	ideal			158	209	532	455	324	563	488
+DellaRosa-2010-417-88	88	Illusione	illusion			205	226	537	427	338	594	488
+DellaRosa-2010-417-89	89	Immortalità	immortality			132	200	387	430	325	630	650
+DellaRosa-2010-417-90	90	Impazienza	impatience			306	287	479	400	448	500	515
+DellaRosa-2010-417-91	91	Importanza	importance			256	205	586	310	312	559	500
+DellaRosa-2010-417-92	92	Imprecazione	curse	8	CURSE	447	385	407	465	379	262	494
+DellaRosa-2010-417-93	93	Impulso	impulse			300	300	437	452	338	506	463
+DellaRosa-2010-417-94	94	Impunità	impunity			190	213	276	550	243	558	610
+DellaRosa-2010-417-95	95	Incantesimo	spell			284	300	447	245	385	506	563
+DellaRosa-2010-417-96	96	Incertezza	uncertainty			229	292	436	415	450	470	553
+DellaRosa-2010-417-97	97	Incongruenza	incongruity/inconsistency			197	220	385	541	278	567	585
+DellaRosa-2010-417-98	98	Incredulità	incredulity			211	303	426	430	406	563	425
+DellaRosa-2010-417-99	99	Indifferenza	unconcern			294	242	468	445	418	506	524
+DellaRosa-2010-417-100	100	Indipendenza	independence			235	288	593	475	438	529	553
+DellaRosa-2010-417-101	101	Indiscrezione	indiscretion			188	191	395	556	303	562	616
+DellaRosa-2010-417-102	102	Indolenza	indolence			226	300	274	583	245	542	615
+DellaRosa-2010-417-103	103	Inefficenza	inefficiency			245	274	418	521	345	515	563
+DellaRosa-2010-417-104	104	Inesperienza	inexperience			232	220	427	458	355	550	556
+DellaRosa-2010-417-105	105	Infallibilità	infallibility			138	200	273	485	250	586	635
+DellaRosa-2010-417-106	106	Infanzia	childhood			321	456	640	425	521	305	479
+DellaRosa-2010-417-107	107	Infelicità	unhappiness			281	353	509	385	450	479	428
```

Comment:
Similar case to the calm/calmness above. I think if this is not mapped to `UNHAPPY`, calm/calmness should also be unmapped.